lib/engine/round/stock.rb

```diff
@@ -83,7 +83,7 @@ def finish_round
       end
 
       def corporations_to_move_price
-        @game.corporations.select(&:floated?)
+        @game.corporations.select { |c| c.floated? && c.type != :minor }
```

Comment:
In most games, minors' stock prices do not move at the end of the stock round. Make this the default, rather than have most games need to override.